packages/plugin-ext/src/plugin/node/debug/plugin-debug-adapter-tracker.ts

```diff
@@ -0,0 +1,85 @@
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+// tslint:disable: no-any
+
+import * as theia from '@theia/plugin';
+
+export class CompositeDebugAdapterTracker implements theia.DebugAdapterTracker {
```

Comment:
please name files after classes or other way around, it's something which we have in coding guidelines for long time